pkg/reaper/podreaper/podreaper.go

```diff
@@ -41,6 +44,11 @@ func Run(args *Args) error {
 		return err
 	}
 
+	if len(fakePods) > 0 && args.K8sConfigPath == "fake" {
```

Comment:
Same for this, all mocking/faking should happen within the test code.

Reply:
👍